x-pack/plugins/transform/server/routes/api/transforms.ts

```diff
@@ -174,14 +183,43 @@ export function registerTransformsRoutes(routeDependencies: RouteDependencies) {
       path: addBasePath('delete_transforms'),
       validate: {
         body: schema.maybe(schema.any()),
+        query: schema.object({
+          /**
+           * Transform Destination Index
+           */
+          deleteDestIndex: schema.maybe(schema.boolean()),
+          deleteDestIndexPattern: schema.maybe(schema.boolean()),
+        }),
```

Comment:
Please don't use inline schema definitions, we need the for generating API documentation. It has to be placed in `schemas` directory